content/android-privacy-policy.md

```diff
@@ -1,11 +1,11 @@
 # Syncthing-Android Privacy Policy
 
-The app syncthing-android does not collect any user information.
+The app syncthing-android values user privacy and thus does not collect any user information.
 
-The app does not store any location data.
+As location data is very sensitive personal information relating to your privacy, the app does not store any location data.
```

Comment:
For location data includes very sensitive personal information related to user privacy, the app does not store any location data.